[2022-02-12 23:27:53,937] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A scheduled__2022-02-11T00:00:00+00:00 [queued]>
[2022-02-12 23:27:53,968] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A scheduled__2022-02-11T00:00:00+00:00 [queued]>
[2022-02-12 23:27:53,968] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 23:27:53,968] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-12 23:27:53,968] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 23:27:54,016] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): training_model_A> on 2022-02-11 00:00:00+00:00
[2022-02-12 23:27:54,024] {standard_task_runner.py:52} INFO - Started process 372 to run task
[2022-02-12 23:27:54,031] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_A', 'scheduled__2022-02-11T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp0hbc8mag', '--error-file', '/tmp/tmp5o4eqr9b']
[2022-02-12 23:27:54,031] {standard_task_runner.py:77} INFO - Job 13: Subtask training_model_A
[2022-02-12 23:27:54,230] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.training_model_A scheduled__2022-02-11T00:00:00+00:00 [running]> on host f9b316006a9e
[2022-02-12 23:27:54,460] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-02-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-11T00:00:00+00:00
[2022-02-12 23:27:54,468] {python.py:175} INFO - Done. Returned value was: 9
[2022-02-12 23:27:54,550] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_A, execution_date=20220211T000000, start_date=20220212T232753, end_date=20220212T232754
[2022-02-12 23:27:54,628] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-12 23:27:54,759] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
